templates/apps/sumo-aws-apps/common-org/common-org-configrecorder-role.yaml

```diff
@@ -0,0 +1,187 @@
+            Statement:
+              - Sid: AllowReadIamActions
+                Effect: Allow
+                Action: 
+                  - iam:GetRole
+                Resource: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/*
+              - Sid: AllowPassRoleForLogGroupRole
+                Effect: Allow
+                Action: iam:PassRole
+                Resource: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/*'
+                Condition:
+                  StringEqualsIfExists:
+                    iam:PassedToService:
+                      - config.amazonaws.com
+                      - config-multiaccountsetup.amazonaws.com
+                      - resource-groups.amazonaws.com
+
+        - PolicyName: config-policy-config
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: AllowConfigActions
+                Effect: Allow
+                Action:
+                  - config:DescribeConfigurationRecorderStatus
+                  - config:DescribeConfigurationRecorders
+                  - config:DeleteDeliveryChannel
+                  - config:PutConfigurationRecorder
+                  - config:DescribeDeliveryChannels
+                  - config:StopConfigurationRecorder
+                  - config:StartConfigurationRecorder
+                  - config:PutDeliveryChannel
+                  - config:DeleteConfigurationRecorder               
+                Resource: '*'
+
+        - PolicyName: resource-groups-policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: ResourceGroupsPolicy
+                Effect: Allow
+                Action:
+                - resource-groups:DeleteGroup
+                - resource-groups:CreateGroup
+                - resource-groups:GetGroup
+                - resource-groups:GetTags
+                - resource-groups:ListGroups
+                - resource-groups:Tag            
+                Resource:
+                  - !Sub 'arn:${AWS::Partition}:resource-groups:*:${AWS::AccountId}:group/*'
+
+  ConfigRecorderRole:
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W28
+            reason: Resource found with an explicit name, this disallows updates that require replacement of this resource
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Ref ConfigRecorderRoleName
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - config.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWS_ConfigRole"
+      Policies:
+        - PolicyName: root
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
```

Comment:
Why does Config need S3 bucket permissions ?

Reply:
If member account not enable AWSConfg or Not connect Bucket for recorder. we will use it. it can use for AWS config logs collection.  I am using S3 bucket present in log archival account with cloudtrail